Overview.src.html

```diff
@@ -3557,9 +3520,41 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 is itself associated with <span title=concept-Request-request>request</span>'s
 <span title=concept-request-header-list>header list</span>.
 
-<p>A <code>Request</code> object's <span title=concept-Body-body>body</span> is its
-<span title=concept-Request-request>request</span>'s
-<span title=concept-request-body>body</span>.
+<p>A <code>Request</code> object also has an associated
+<dfn title=concept-Request-disturbed-flag>disturbed flag</dfn> which is initially unset.
+
+<p>A <code>Request</code> object's <span title=concept-Body-disturbed>disturbed</span> predicate
+returns true if the associated <span title=concept-Request-request>request</span>'s
+<span title=concept-request-body>body</span> is not null and the associated
+<span title=concept-Request-disturbed-flag>disturbed flag</span> is set.
+
+<p>A <code>Request</code> object's <span title=concept-body-consume-body>consume body</span>
+algorithm, which given a <var>type</var>, runs these steps:
+
+<ol>
+ <li><p>If this <code>Request</code> is <span title=concept-Body-disturbed>disturbed</span>, return
+ a new promise rejected with a <code>TypeError</code>.
+
+ <li><p>Set <span title=concept-Request-disturbed-flag>disturbed flag</span>.
+
+ <li><p>Let <var>p</var> be a new promise.
+
+ <li>
+  <p>Run these substeps <span data-anolis-spec=html>in parallel</span>:
+  <ol>
+   <li><p>Let <var>bytes</var> be <span title=concept-Request-request>request</span>'s
```

Comment:
Shouldn't this do a bit more? E.g., read/consume the body somehow since it's a stream?